src/backend/graphics/egl.rs

```diff
@@ -0,0 +1,662 @@
+//! Common traits and types for opengl rendering on graphics backends
+
+/// Large parts of the following file are taken from
+/// https://github.com/tomaka/glutin/tree/master/src/api/egl at commit
+/// `044e651edf67a2029eecc650dd42546af1501414`
+///
+/// It therefor falls under glutin's Apache 2.0 license
+/// (see https://github.com/tomaka/glutin/blob/master/LICENSE)
```

Comment:
"therefor**e**"

Also I guess this should be good enough. But to be safe, let's ask @tomaka his opinion on the matter.